lib/console_yasttest.pm

```diff
@@ -17,6 +17,14 @@ sub post_fail_hook() {
     # only upload if save_y2log succeeded
     if (!script_run "save_y2logs $fn") {
         upload_logs $fn;
+    } else {
+        # there is a severe problem, e.g. could be bsc#985850 or bsc#990384 so
+        # save more, let's hope there is enough memory for intermediate
+        # storage
+        # TODO just assuming that '/dev/vda2' is the root device here
+        $fn = '/dev/shm/vda2_brfs_debug_tree';
+        assert_script_run "btrfs-debug-tree /dev/vda2 &> $fn";
```

Comment:
pushed the tiny change to master and want to trigger some tests on osd for the night :-)